internal/proto/local/controller/api/resources/authmethods/v1/auth_method.proto

```diff
@@ -67,42 +67,42 @@ message OidcAuthMethodAttributes {
 	// an immediate need to reauthenticate.
-	google.protobuf.Int32Value max_age = 60 [json_name="max_age", (custom_options.v1.mask_mapping) = {this:"attributes.max_age" that: "MaxAge"}];
+	google.protobuf.Int32Value max_age = 60 [json_name="max_age", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.max_age" that: "MaxAge"}];
 
 	// The signing algorithms allowed for an oidc auth method
-	repeated string signing_algorithms = 70 [json_name="signing_algorithms", (custom_options.v1.mask_mapping) = {this:"attributes.signing_algorithms" that: "SigningAlgs"}];
+	repeated string signing_algorithms = 70 [json_name="signing_algorithms", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.signing_algorithms" that: "SigningAlgs"}];
 
 	// The url prefixes used by the OIDC provider for call back url, the token
 	// retrieval url, and the final redirect url used in the authentication flow
 	// for this auth method.
-	google.protobuf.StringValue api_url_prefix = 80 [json_name="api_url_prefix", (custom_options.v1.mask_mapping) = {this:"attributes.api_url_prefix" that: "CallbackUrls"}];
+	google.protobuf.StringValue api_url_prefix = 80 [json_name="api_url_prefix", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.api_url_prefix" that: "CallbackUrls"}];
 
 	// Output only. The callback url used by the OIDC provider in the authentication flow. This will have
 	// boundary's callback api path appended too them.
 	string callback_url = 90 [json_name="callback_url"];
 
 	// certificates are optional PEM encoded x509 certificates that can be
 	// used as trust anchors when connecting to an OIDC provider.
-	repeated string certificates = 100 [json_name="certificates", (custom_options.v1.mask_mapping) = {this:"attributes.certificates" that: "Certificates"}];
+	repeated string certificates = 100 [json_name="certificates", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.certificates" that: "Certificates"}];
 
 	// The audience claims for this oidc typed auth method.
-	repeated string allowed_audiences = 110 [json_name="allowed_audiences", (custom_options.v1.mask_mapping) = {this:"attributes.allowed_audiences" that: "AudClaims"}];
+	repeated string allowed_audiences = 110 [json_name="allowed_audiences", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.allowed_audiences" that: "AudClaims"}];
 
 	// Input only. This is only valid in an update request and should not be used
 	// unless the oidc provider's config is incorrectly set and is stopping the
 	// activation of this auth method.
-	bool override_oidc_discovery_url_config = 120 [json_name="override_oidc_discovery_url_config"];
+	bool override_oidc_discovery_url_config = 120 [json_name="override_oidc_discovery_url_config", (custom_options.v1.generate_sdk_option) = true];
```

Comment:
I think it is ok to iterate on this name a bit until we are more happy with it.  I think removing the `_oidc` portion of it might be good.  I think we can still make progress in parallel regardless of the naming issue.